### DISCUSSION TITLE
Is detectron2 abandoned project ?

### DISCUSSION THREAD

**Discussion by unrue** [facebookresearch/detectron2#5052] [General]
Latest version of detectron2 is almost two years ago. Is the project still active or is death? Thanks.

**Comment by ravijo** [comment]
It seems so. Though there are minor updates, such as supporting dependencies-related issues, etc., I don't see any activities in the [discussions](https://github.com/facebookresearch/detectron2/discussions/) and [issues](https://github.com/facebookresearch/detectron2/issues) section.

**Reply by unrue** [reply]
Are you sure? I see tickets opened 3 days ago.

**Reply by ravijo** [reply]
@unrue Sorry, I am not sure. But I still don't see any activities as I said before.
